### PR TITLE
npm audit fix; Update vulnerabilities

### PR DESCRIPTION
**Pull request by apeinsip**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,10 +28,10 @@
   "homepage": "https://github.com/malykhinvi/node-mongodb-changelog",
   "dependencies": {
     "kerberos": "0.0.23",
-    "mongodb": "2.2.31"
+    "mongodb": "^3.5.0"
```
Comment (malykhinvi) [inline package.json]: I very appreciate your input, but apparently this version has a bit different API which is not compatible with current implementation of `node-mongodb-changelog`.

https://travis-ci.org/malykhinvi/node-mongodb-changelog/builds/637407329?utm_source=github_status&utm_medium=notification

File: .gitignore [hunk 2]
```diff
@@ -14,5 +14,5 @@ results
 npm-debug.log
 node_modules
 .idea
-
+*.iml
```
Comment (malykhinvi) [inline .gitignore]: Looks like this change does not relate to vulnerabilities fix. Am I right that `*.iml` files used by JetBrains IDE? In that case having `.idea` directory in gitignore should be enough.

Comment (apeinsip) [reply .gitignore]: this is intellij - has to set gitignore

File: package.json (hunk 1 above)
Comment (apeinsip) [reply package.json]: Do you make these changes? Go to Mongo version 3?

Comment (hendrik-scholz) [reply package.json:31]: Can you please check whether the changes I've made in https://github.com/hendrik-scholz/node-mongodb-changelog are the changes you are looking for. Please keep in mind, that I'm still developing on my fork of this repo. I'll create a pull request when I'm done with the changes I'd like to make.

**Comment by malykhinvi** [comment]
It has been a while since the last upgrade of a `mongodb-changelog`. Now I think I have a bandwidth to contribute to that library. I'm afraid a major upgrade of `mongodb` without a major version bumping would break dependant packages.
As I have to ship a new major version, I would like to combine these changes with other updates. For example it is time to drop support for legacy nodejs versions. See a milestone I created for these updates https://github.com/malykhinvi/node-mongodb-changelog/milestone/1
Guys I appreciate your input, but I'm closing that PR in favour of upcoming v3 changes.

**Pull request closed without merging**